src/SDL12_compat.c

```diff
@@ -85,6 +85,67 @@
 #define SDL20_zerop(x) SDL20_memset((x), 0, sizeof(*(x)))
 #define SDL_ReportAssertion SDL20_ReportAssertion
 
+/* From SDL2.0's SDL_bits.h: a force-inlined function. */
+#if defined(__WATCOMC__) && defined(__386__)
+extern _inline int _SDL20_bsr_watcom (Uint32);
+#pragma aux _SDL_bsr_watcom = \
```

Comment:
This needs renaming to `_SDL20_bsr_watcom`

Reply:
Whoops! Fixed now.